plugins/available/cmd-duration.plugin.bash

```diff
@@ -0,0 +1,46 @@
+cite about-plugin
```

Comment:
I'm wondering whether this needs to be a dedicated plugin - maybe we could add this functionality to the `base` theme so that it becomes available to all themes?

Not sure about the initialization logic, i.e. when to define the `PS0` variable - this might need to be triggered from each theme individually.

Reply:
A plugin that can be enable / disabled by the user seem like a good choice for configuration.

Themes can be updated a simple block of code to use the feature:
```bash
command_duration=
if _command_exists _command_duration; then
  command_duration=$(_command_duration)
fi
```

Reply:
I think this file should not be a plugin and instead reside in `base.theme.bash` or `cmd_duration.theme.bash` and loaded in `bash_it.sh:65`  
This is the way we currently handle new features in themes. I think a flag from the user can maybe control whether they want it or disable it in their main `.bashrc` file.  

Lemme know what you think @BarbUk

Reply:
Thanks for the feedback.

> This is the way we currently handle new features in themes.

Seems fair to me.

> I think a flag from the user can maybe control whether they want it or disable it in their main .bashrc file.

Yes, this was the main argument to make it a plugin. 
I want users to make the decision to enable it if they want or not.

2e8ad81e127ca27d869390ee2a35bfd187a1cf70 remove the plugin and make it a theme feature.
It also introduce a `BASHIT_COMMAND_DURATION` flag that can be set to true to enable the feature.

Reply:
> It also introduce a BASHIT_COMMAND_DURATION flag that can be set to true to enable the feature.

The flag was renamed to BASH_IT_COMMAND_DURATION

Reply:
@NoahGorny @nwinkler Does the migration to a theme feature seems good to you ?